prime-router/metadata/schemas/upload-covid-19.schema

```diff
@@ -358,6 +359,8 @@ elements:
       cardinality: ONE 
       csvFields: [{name: testing_lab_name}]
 
+    - name: testing_lab_name_type_code
```

Comment:
It definitely doesn't hurt to have this here but I think as long as the element is present in the COVID-19 schema or whatever schema the receiver is using you get the same result whether this is here or not

Reply:
If this is true, I think there's some documentation that needs to change somewhere because I swear I read that "based on" requires you to specify elements you want to include.